README.md

```diff
@@ -17,26 +17,42 @@ $ pipenv install
 The site uses [elsa](https://github.com/pyvec/elsa).
 
 - Installation: `pipenv install --dev`
-- Development server: `pipenv run serve`
+- Download data from external sources: `pipenv run build`
 - Tests: `pipenv run test`
+- Development server: `pipenv run serve`
 
 ### Data and tests
 
 The site is just a single HTML page rendered on top of some static data.
-However, the data can get quite complex and most of the texts need to be
-translated into two languages.
+However, some of the data come from external sources, the data can get quite
+complex, and most of the texts need to be translated into two languages.
 
 The data is stored in multiple YAML files. When these are read, whenever
 an object has just `cs` and `en` properties, it is treated as a "translated text"
 and the property corresponding to a currently selected language becomes
 the actual value in place of the object.
 
-Also, to keep the complex structure of the YAML files organized and tested,
-there are schemas written in [JSON Schema](https://spacetelescope.github.io/understanding-json-schema/)
+To keep the complex structure of the YAML files organized and tested,
+there are schemas written in [JSON Schema](https://json-schema.org/understanding-json-schema/)
 ([spec](http://json-schema.org/)). In tests, the YAML files are validated
 against the schemas. There is also a couple of additional tests to ensure some
 logical rules which cannot be easily expressed by JSON Schema.
 
+### External sources
+
+Some data cannot be stored statically in a YAML file. There is a command
+`pipenv run build`, which downloads them from external sources and generates
+respective static YAML files. This is a separate step, which needs to be done
+before developing or deploying the site, otherwise it won't work properly.
+
+### Members
+
+Pyvec members are tracked in an internal Google Spreadsheet. The future
+intention is to have the list of members public, but we're not there yet (GDPR).
```

Comment:
The used CSV exported from the spreadsheet with all the members is public. I don't understand this part.

Reply:
It is public to those reading source code of this site. It is not public on the web for mere mortals. It seemed as get things done & good enough solution for now.

Reply:
The sentence means public in a sense anyone could go to pyvec.org, do clickity-click and read who's member of Pyvec.

Reply:
They could then be stored in a yaml in this repo in a same security by obscurity way. Definitively not a good way to mention GDPR with this :scream:

Reply:
Ok lets say I want to have this prepared for the time when we do have paperwork in place and can have the list public. For now, the people in the list gave me an unwritten and unofficial consent. Having these public is with no risk - they are public even without this PR. But I don’t want to add new people to the list until the paperwork is done.

Reply:
(also all the people who are in the board are publicly available at justice.cz - basically it is the whole essence of board to have the members public, so other entities interacting with Pyvec can e.g. verify that documents signed are being signed by a person who can sign them on behalf of Pyvec)